Assets/Scripts/Boss/BossPhaseOne.cs

```diff
@@ -29,6 +31,14 @@ public void ShootPrimaryShot()
             StartCoroutine(CountCooldown(primaryBulletCooldown));
         }
 
+        public void ShootFollowStraightShot()
+        {
+            if (!_canShootExtra) return;
+            BossShoot(typeOfShoots[3], spawnPoints[4]);
+            BossShoot(typeOfShoots[3], spawnPoints[5]);
```

Comment:
Later, change to weapon-specific spawn point vectors ou a dictionary for spawn point and bullets